lib/util.lua

```diff
@@ -65,27 +67,98 @@ function windowsCompile(ctx)
         print("size:\t" .. size .. " bytes")
     end
 
-    --local exitCode = os.execute('msiexec /quiet /a "' .. qInstallFile .. '" TargetDir="' .. qInstallPath .. '"')
-    print("Installing python, please wait patiently for a while, about two minutes.")
-    local exitCode = os.execute(qInstallFile .. ' /quiet InstallAllUsers=0 PrependPath=0 TargetDir=' .. qInstallPath .. ' /log ' .. qInstallLog)
-    if exitCode ~= 0 then
-        -- show install log
-        local logFile = io.open(qInstallLog, "r")
-        if logFile then
-            local contents = logFile:read("*a")
-            logFile:close()
-            print("=====================================")
-            print(qInstallLog)
-            print(contents)
-            print("=====================================")
-            print("Please uninstall python manually and try again.")
+    -- Extract
+    print("Extracting installer...")
+    local wixBin = os.getenv("USERPROFILE") .. '\\.version-fox\\plugin\\python\\bin\\WiX\\dark.exe'
```

Comment:
You can use `RUNTIME.pluginDirPath` to get the path of plugin home.

https://github.com/version-fox/vfox-plugin-template/blob/fddd1526732f6e2552c2725a63d4d56b5de34cc0/Injection.lua#L15

Reply:
resolved.